src/litserve/server.py

```diff
@@ -472,3 +472,100 @@ def setup_auth(self):
         if LIT_SERVER_API_KEY:
             return api_key_auth
         return no_auth
+
+
+@asynccontextmanager
+async def manage_lifespan(app: FastAPI, servers: List[LitServer]):
+    """Context manager to handle the lifespan events of multiple FastAPI servers."""
+    # Start lifespan events for each server
+    lifespans = [server.lifespan(server.app) for server in servers]
+    for lifespan in lifespans:
+        await lifespan.__aenter__()
```

Comment:
The following would be more Pythonic using [AsyncExitStack](https://docs.python.org/3/library/contextlib.html#contextlib.AsyncExitStack)

```python
from contextlib import AsyncExitStack

async with AsyncExitStack() as stack:
    connections = [await stack.enter_async_context(lifespan) for lifespan in lifespans]
```